demos/common/python/requirements_openvino.txt

```diff
@@ -0,0 +1 @@
+openvino-dev
```

Comment:
why do you need dev package?

dev package contains only tools (mo, pot, ac, model downloader), the runtime itself distributed in separated package.

Also it is not good idea to put openvino to requirements as model api aligned with new version, which is not published to pypi as main release, if user executes
`pip install openvino-dev`, it get tools for ov 2021.4.2

Reply:
> Also it is not good idea to put openvino to requirements  

Do you mean it is better to delete the `requirements_openvino.txt` at all? I think it is actually will be useless if the Python Model API is included to openvino-dev package

For the current state, what the correct way to install openvino runtime? I added these requirements file for case if one installs pure Model API package without installed openvino.  
 It is almost impossible case, but if we delete the `requirements_openvino.txt`, it means model API does not rely on OpenVINO if `OpenvinoAdapter` is chosen and it is not true. So this dependency should be highlighted somehow. 

Probably the reference to OpenVINO repository will be enough, if the runtime is not published to pypi as main release?

Reply:
probably you need to write in docs as prerequisites. When api become a part of openvino-dev, then we will have 2 paths for installation - as part of dev package (here everything is ok, no need additional deps install) and from source in instruction where you need to install model api from source you need mention that openvino runtime python package should be installed (from source code, from pypi e.g.). something similar written in https://github.com/openvinotoolkit/open_model_zoo/pull/3156

For now, better to remove this requirements list as it does not pass pip conflicts checking ci job for publishing submodule in ov...(

Reply:
I removed the txt file with openvino-dev and added the `Prerequisites` section.